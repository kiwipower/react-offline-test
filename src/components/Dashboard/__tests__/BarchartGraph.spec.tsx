import * as React from "react";
import { shallow } from "enzyme";
import BarchartGraph from "../BarChartGraph";

const generationmix = [
  {
    fuel: "biomass",
    perc: 4.8
  },
  {
    fuel: "coal",
    perc: 2.5
  }
];

describe("<BarchartGraph />", () => {
  it("should not render", () => {
    const component = shallow(<BarchartGraph energy={null} />);
    expect(component.type()).toEqual(null);
  });

  it("should render", () => {
    const energy = {
      generationmix,
      from: "2019-08-12T13:00Z",
      to: "2019-08-12T12:30Z"
    };
    const component = shallow(<BarchartGraph energy={energy} />);
    expect(component).toMatchSnapshot();
  });
});
