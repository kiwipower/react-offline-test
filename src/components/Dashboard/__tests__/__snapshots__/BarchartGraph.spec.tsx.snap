// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BarchartGraph /> should render 1`] = `ShallowWrapper {}`;
